cmd/admission/main.go

```diff
@@ -63,7 +64,9 @@ func main() {
 		glog.Fatalf("Unable to build k8s config: %v\n", err)
 	}
 
-	admissioncontroller.KubeBatchClientSet = app.GetKubeBatchClient(restConfig)
+	admissioncontroller.VolcanoClientSet = app.GetVolcanoClient(restConfig)
+
+	servePods(config)
```

Comment:
Why donot you keep consistent with L54-55?

Reply:
because pods need client and parameter。